src/frecency.rs

```diff
@@ -57,6 +57,16 @@ where
         }
     }
 
+    pub fn insert(&mut self, key: T) {
```

Comment:
frecency has decent unit test coverage; you could add a unit test for the insert-once behavior of this.